src/test/java/TopDaoTests.java

```diff
@@ -0,0 +1,145 @@
+import TTT.databaseUtils.TopDAO;
+import TTT.peaksAndSummitsHandler.Top;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class TopDaoTests {
```

Comment:
Those tests do not seem to be able to run independantly, rather all at once/in order. Please introduce tests in such a way that they're not dependant on each other.

Reply:
Ok, do some changes, please check it.